website/content/docs/plugins/plugin-development.mdx

```diff
@@ -113,6 +113,25 @@ instead implement the
 [`PluginVersioner`](https://github.com/hashicorp/vault/blob/sdk/v0.6.0/sdk/logical/logical.go#L150-L154)
 interface directly.
 
+## Plugin logging
+
+Auth and secrets plugins based on `framework.Backend` from the SDK can take
+advantage of the SDK's [default logger](https://github.com/hashicorp/vault/blob/fe55cbbf05586ec4c0cd9bdf865ec6f741a8933c/sdk/framework/backend.go#L437).
```

Comment:
Any reason to not link to the line on `main`?

Reply:
That logger used in main is only used to log an error serving the plugin. We have to instantiate a new logger there because if we couldn't serve the plugin then we don't have a logger to use.